WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -865,7 +808,7 @@ private void cancelDomainCreditsCheckProgressDialog() {
             mCheckingDomainCreditsProgressDialog.cancel();
         }
     }
-
+    @SuppressWarnings("deprecation") // Progress Dialog Deprecation on API 26
```

Comment:
Reviewer Note : 

Do we have any issue which says of migrating the deprecated Progress Dialog to ProgressBar/Notification? If so then should I comment issue number over here for ease in understanding the issue?

Reply:
Reverting this due to not related to context of the PR.